test/environment.rb

```diff
@@ -47,15 +47,16 @@ def self.setup
       File.expand_path('../database.ci.yml', __FILE__)
     end
 
-    # This only affects postgres
-    # the :ruby code path will get tested in mysql and sqlite3
-    Ancestry.default_update_strategy = :sql
-
     # Setup database connection
     config = YAML.load_file(filename)[db_type]
     ActiveRecord::Base.establish_connection config
     begin
       ActiveRecord::Base.connection
+
+      # This only affects postgres
+      # the :ruby code path will get tested in mysql and sqlite3
+      Ancestry.default_update_strategy = :sql if ActiveRecord::Base.connection.adapter_name.downcase == "postgresql"
```

Comment:
Think we already know the db type here, lets do:

```suggestion
      # the :ruby code path will get tested in mysql and sqlite3
      Ancestry.default_update_strategy = :sql if dbtype == "pg"
```